src/analysis.jl

```diff
@@ -0,0 +1,19 @@
+"""
+    A collection of analytic tools to evaluate CellML models
+"""
+
+function simplify_systems(systems)
+    for x in systems
+        sys = last(x)
+        print("simplifying $(sys.name)")
```

Comment:
actually eventually lets make stdout stuff a verbose flag

Reply:
I agree. However, this function is a debug function that is called manually. It is not called by any other function.